examples/favorites_advanced/redux_favorites_advanced/test/feature_puppy/search/puppy_list_test.mocks.dart

```diff
@@ -0,0 +1,72 @@
+// Mocks generated by Mockito 5.0.3 from annotations
+// in redux_favorite_advanced_sample/test/feature_puppy/search/puppy_list_test.dart.
+// Do not manually edit this file.
+
+import 'dart:async' as _i5;
+
+import 'package:favorites_advanced_base/src/models/puppy.dart' as _i2;
+import 'package:favorites_advanced_base/src/repositories/puppies_repository.dart'
+    as _i4;
+import 'package:favorites_advanced_base/src/utils/enums.dart' as _i6;
+import 'package:image_picker_platform_interface/src/types/picked_file/unsupported.dart'
```

Comment:
@GoranPrime 

```
Snezhana Romanova, Fri 9:31 AM
Hi, guys! When you start using Mock, there is something you need to correct manually. Everywhere you are mocking PuppiesRepository, in your autogenerated file you have to change the end of one of imports like that:
autogenerated :
import 'package:image_picker_platform_interface/src/types/picked_file/unsupported.dart'
correct the end:
unsupported.dart => picked_file.dart
Be carefull - each time you run "flutter pub run build_runner build" it will rollback all your corrections
```

Reply:
Corrected